Makefile

```diff
@@ -1,11 +1,12 @@
 GIT_COMMIT?="$(shell git rev-parse HEAD | head -c 7)"
 NAME_POSTFIX?="$(shell docker ps -a | wc -l | xargs)"
 TESTER_TAG?="noobaa-tester"
-SERVER_TAG?="noobaa-server"
+NOOBAA_TAG?="noobaa-server"
```

Comment:
the image is also for agents. maybe change the tag from `noobaa-server` to `noobaa`?